dataactcore/scripts/load_duns.py

```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 
 
-def process_duns_dir(sess, historic, local, benchmarks=None, metrics=None):
+def process_duns_dir(sess, historic, local, benchmarks=None, metrics=None, force_reload=None):
```

Comment:
Normally we use `force_reload` to require it to reload the data even if we have no changes in the file. This is normally just a flag. I think we should keep to that standard and name this something like `reload_date` instead